force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -41,8 +45,13 @@ public with sharing class FlowControlLoopsCollections {
      * @param age The age of the person.
      * @return true if the person is eligible to vote, false otherwise.
      */
-    public static Boolean votingEligibility(Integer age) {
-        return null; // Replace null with the variable you used to store the result
+    public static Boolean votingEligibility(Integer age) { 
+        if (age >= 18){
+            return true;
+        } else {
+            System.debug('Underaage voting is ineligible.');
```

Comment:
Try to do this with ternary operator and remove debugs from the final product unless absolutely necessary